docs/Examples.md

```diff
@@ -57,6 +74,13 @@ go run examples/http-sec/main.go
 
 ## [Kafka Service](../examples/kafka/main.go)
 
+---
```

Comment:
I found useful making this comment because by simply trying to start both services they will fail as they attempt to listen the same port